server/routerlicious/packages/lambdas/src/alfred/interfaces.ts

```diff
@@ -0,0 +1,70 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IEvent } from "@fluidframework/common-definitions";
+import { IClient, IConnected } from "@fluidframework/protocol-definitions";
+
+/**
+ * Connection details of a client
+ */
```

Comment:
@znewton @hedasilv @pradeepvairamani do you have some context to add here?

Reply:
I think the added docs for the 3 inner properties is enough, IMO